mods_check.py

```diff
@@ -1,35 +1,40 @@
 #!/usr/bin/env python3
 
+from bungie_api import BungieApi
+from typing import Iterable, Tuple, Any
+from api_types import Membership
 from item_info import Item
-from DestinyEnums import D2Vendors
+from DestinyEnums import D2Vendors, DestinyItemType
 
 
-def banshee_has_new_mod(bungie_api, membership_id):
+def banshee_has_new_mod(bungie_api: BungieApi, membership: Membership) -> Tuple[Iterable[str], Iterable[Item]]:
     # get character info
-    first_character_id = bungie_api.get_profile(membership_id)
+    first_character_id = bungie_api.get_profile(membership)
     # get items vendor is selling for that character
-    banshee_vendor_info = bungie_api.get_vendor(
-        membership_id, first_character_id, D2Vendors.Banshee.value
-    )
-    item_sales = banshee_vendor_info["Response"]["sales"]["data"]
     item_list = []
-    for x in item_sales.values():
-        item_hash = x["itemHash"]
-        item_response = bungie_api.get_item_definition(item_hash)
-        if "collectibleHash" in item_response["Response"].keys():
-            item = Item(
-                item_response["Response"]["collectibleHash"],
-                item_response["Response"]["displayProperties"]["name"],
-            )
-            item_list.append(item)
+    mod_vendors = [D2Vendors.Banshee, D2Vendors.Ada]
+    for vendor in mod_vendors:
+        vendor_info = bungie_api.get_vendor(
+            membership, first_character_id, vendor.value
+        )
+        item_sales = vendor_info["Response"]["sales"]["data"]
+        for x in item_sales.values():
+            item_hash = x["itemHash"]
+            item_response = bungie_api.get_item_definition(item_hash)
+            if "collectibleHash" in item_response["Response"].keys() and "itemType" in item_response["Response"].keys() and item_response["Response"]["itemType"] == DestinyItemType.Mod.value:
+                item = Item(
+                    item_response["Response"]["collectibleHash"],
+                    item_response["Response"]["displayProperties"]["name"],
+                )
+                item_list.append(item)
```

Comment:
small nit that it's probably better to call this mod list since we're filtering it down now